src/ucc/api/ucc.h

```diff
@@ -659,6 +660,48 @@ typedef struct ucc_oob_coll {
 
 typedef ucc_oob_coll_t ucc_context_oob_coll_t;
 typedef ucc_oob_coll_t ucc_team_oob_coll_t;
+
+/**
+ *
+ *  @ingroup UCC_CONTEXT_DT
+ */
+typedef enum  {
+    UCC_MEM_CONSTRAINT_SYMMETRIC   = UCC_BIT(0),
```

Comment:
@wfaderhold21 Do you currently use this constraint to enable/disable the non-symmetric flow?